e2e/teardown.ts

```diff
@@ -0,0 +1,13 @@
+import { deleteFork } from './utils/tenderly';
+
+async function globalTeardown() {
```

Comment:
no action required, but lets try to keep arrow syntax for consistency in the future